core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -67,14 +67,15 @@ class Controller(
         // handleRejections wraps the inner Route with a logical error-handler for
         // unmatched paths
         handleRejections(customRejectionHandler) {
-            super.routes ~ apiv1.routes
+            super.routes ~ apiv1.routes ~ apiv2.routes
         }
     }
 
     logging.info(this, s"starting controller instance ${instance}")
 
     /** The REST APIs. */
     private val apiv1 = new RestAPIVersion_v1(config, context.system, logging)
+    private val apiv2 = new RestAPIVersion_v2(config, context.system, logging)
```

Comment:
Currently seeing a pause when calling which ever route is instantiated second here...